src/reconcile.rs

```diff
@@ -0,0 +1,81 @@
+use crate::client::Client;
+use crate::error::Error;
+use k8s_openapi::api::core::v1::Pod;
+use kube::api::{ListParams, Meta, ObjectMeta};
+use kube_runtime::controller::ReconcilerAction;
+use std::time::Duration;
+
+pub type ReconcileResult<E> = std::result::Result<ReconcileFunctionAction, E>;
+
+pub fn create_requeuing_reconciler_action(secs: u64) -> ReconcilerAction {
+    ReconcilerAction {
+        requeue_after: Some(Duration::from_secs(secs)),
+    }
+}
+
+pub fn create_non_requeuing_reconciler_action() -> ReconcilerAction {
+    ReconcilerAction {
+        requeue_after: None,
+    }
+}
+
+#[derive(PartialEq)]
```

Comment:
Yeah I always try to only derive the mininum amount of needed traits and Eq wasn't needed so far. Happy to add it if you think it makes sense.

Reply:
I'd usually agree, but in this specific case it might set people to wondering if there is a reason why Eq is not possible - at least it got me thinking..